cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/MockCruiseControl.java

```diff
@@ -308,6 +324,26 @@ public void setupCCRebalanceResponse(int pendingCalls, int responseDelay, Cruise
                                 .withBody(jsonVerbose)
                                 .withHeaders(header("User-Task-ID", REBALANCE_NO_GOALS_VERBOSE_RESPONSE_UTID))
                                 .withDelay(TimeUnit.SECONDS, responseDelay));
+
+        // Rebalance response with no goals set - without verbose parameter
+        JsonBody jsonWithoutVerbose = new JsonBody(ReadWriteUtils.readFileFromResources(getClass(), "/" + CC_JSON_ROOT + "CC-Rebalance-no-goals.json"));
+
+        server
+                .when(
+                        request()
+                                .withMethod("POST")
+                                .withQueryStringParameter(Parameter.param(CruiseControlParameters.JSON.toString(), "true"))
+                                .withQueryStringParameter(Parameter.param(CruiseControlParameters.DRY_RUN.toString(), "true|false"))
+                                .withQueryStringParameter(buildBrokerIdParameter(endpoint))
```

Comment:
ditto as above

Reply:
renaming the method should fix this issue

Reply:
I will investigate and try to find a way

Reply:
@ppatierno Hi, would it be fine if I keep the one where I have removed the `verbose` parameter? That way I can remove to exactly same mocks. I think of that because ieven if we have `verbose` parameter as part of the query string -> I see it as either this `                                .withQueryStringParameter(Parameter.param(CruiseControlParameters.VERBOSE.toString(), "true|false"))
` or this `                                .withQueryStringParameter(Parameter.param(CruiseControlParameters.VERBOSE.toString(), "false"))`, which means that verbose is not really required since we have another case which covers verbose response

Reply:
I think removing verbose as it's expected now "true|false" should be ok but then where do we have another case covering the query with verbose? We need cases like that because all the other CC endpoints support "verbose" as part of the query string.

Reply:
You mean when verbose is set `true`? I can definitely add that case where missing